### ISSUE TITLE
Escaping of special characters while serialization and de-serialization 

### ISSUE THREAD

**Issue by jas-singh14** [c-rack/cbor-java#67]
How special characters in JSON strings are handled by c-rack library while serialization and de-serialization?
Also is the behaviour same for all the CBOR libraries that conform to RFC 7049?
Please clarify.

**Comment by jas-singh14** [comment]
Doesn't this library converts from JSON to CBOR and vice versa?

**Comment by c-rack** [comment]
No, it does not. It encodes and decodes from Java types to CBOR and vice versa.
